### DISCUSSION TITLE
Add option to set cue out manually (and eventually cue in)

### DISCUSSION THREAD

**Discussion by BornKillaz** [Moonbase59/autocue#25] [Ideas]
It would be nice to have an option to add a parameter for manually set cue out and eventually cue in also.

This was extremely useful if for example, a pre recorded show that is set to air 45 minutes, but the file has 48 minutes long. So, in setting the cue out parameter, we could explicity set it to be 45 minutes long. This way we don't have to rely on other software (like azuracast force to interrupt other playlist).

And since cue_file is explicity focused on cue, then, it makes sense that you can manually add this type of parameter. And of course, make it recognizable by liquidsoap / azuracast.

What do you think?

**Comment by Moonbase59** [comment]
Not currently planned. It’s an _auto_-cue after all.

Setting cue-points (and fades, which you might also need) is better done using…
- a simple _tag editor_ (yes, you can do that, and Liquidsoap will pick it up!)
- _annotations_ in Liquidsoap, which have highest priority and could even override file tags.
- the _Visual Cue Editor_ or _Advanced tab_ in AzuraCast.
